plenoptic/tools/optim.py

```diff
@@ -1,30 +1,28 @@
-"""tools related to optimization
-
-such as more objective functions
+"""Tools related to optimization such as more objective functions.
 """
 import torch
-from typing import Union
+from torch import Tensor
+from typing import Optional, Tuple
 import numpy as np
 
 
-def set_seed(seed: Union[int, None] = None):
+def set_seed(seed: Optional[int] = None) -> None:
```

Comment:
why `Optional[int]` over `Union[int, None]`?

Reply:
They're equivalent https://stackoverflow.com/a/51710151 . Code bases I've worked in mainly go with Optional[whatever] because it's cleaner.

Reply:
sounds good, that's also what the sciware typing session recommended